src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
+	register "sata_ports_enable[1]" = "1" # M.2 2280
+	register "sata_ports_dev_slp[0]" = "1"
+	register "sata_ports_dev_slp[1]" = "1"
```

Comment:
DEVSLP pins are not connected on this board.

Reply:
[Removed](https://github.com/Dasharo/coreboot/pull/576/commits/734785c0a950d3048c11ac982335b31262decdca#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2L8)